tortoise/queryset.py

```diff
@@ -664,6 +679,9 @@ def _make_query(self) -> None:
             self.query._offset = self._offset
         if self._distinct:
             self.query._distinct = True
+        self.resolve_ordering(
```

Comment:
Note that I moved `self.resolve_ordering` to above `self.resolve_filters` to fix a bug in the previous commit. This is now specified twice.

Also, please have a look at `resolve_filters()` as it conditionally adds a default `order_by` at about line 100:
```py
        if has_aggregate and (self._joined_tables or having_criterion or self.query._orderbys):
            self.query = self.query.groupby(
                self.model._meta.basetable[self.model._meta.db_pk_field]
            )
```
Please ensure that when one specifies an group_by manually, that we don't create duplicate group bys.

Reply:
I'm not so clear about that,why there group by ``pk``?

Reply:
It's for when an aggregation runs on a child model (e.g. counting no of children) it needs a group_by on the PK.
Or if you filter on it, it needs to be grouped else "having" won't see it.
The order_by doesn't make sense to me, but if you don' group by on PostgreSQL it fails with syntax error.

I would be happy to not automatically group_by PK at all if a manual group_by is specified, because then you are taking control from us, right?